src/packagedcode/pypi.py

```diff
@@ -542,8 +544,10 @@ def parse(cls, location, package_only=False):
             keywords=get_keywords(project_data),
             parties=get_pyproject_toml_parties(project_data),
             dependencies=dependencies,
+            is_private=is_private,
             extra_data=extra_data,
-            **urls,
+            homepage_url=urls.get('homepage'),
```

Comment:
I don't think the change makes sense here, we should just include the `**urls` like before.